README.md

```diff
@@ -185,4 +185,90 @@ If you're calling a **GET** on `/entities` , you're asking for a list of entitie
 ##### Reasoning
 Using the right HTTP Verb on each resource will make clearer what the resource is made for and what we should expect it to do, receive and return. 
 
+---
+
+#### Use Nouns for endpoints names ( URLs )
+
+> Your resources should be always nouns and no verbs.
+
+##### Examples
+
+###### Good
+
+> Getting an entity
+
+- **GET** `/entities/:id`
+  This retrieve the entity with the Id you pass along the URL.
+
+> Creating an entity:
+
+- **POST** `/entities`
+  This creates an entity with the info you pass along the request body ( We'll get to this later ).
+
+> Deleting an entity
+
+- **DELETE** `/entities/:id`
+  This Deletes the entity with the id you pass along the URL
+
+###### Bad
+
+> Getting an entity
+
+- **GET** `/get/entity?id=:id`
+
+> Creating an entity
+
+- **POST** `/create/entity`
+
+> Deleting an entity
+
+- **DELETE** `/delete_entity/:id`
+
```

Comment:
add an example like `POST /delete_entity/:id` which is a more common mistake than `DELETE /delete_entity/:id`